aries-site/src/pages/components/button.mdx

```diff
@@ -49,6 +49,26 @@ The HTML elements for buttons and links (a.k.a. anchors) describe a very specifi
 - Use buttons when you are performing an action, such as: "submit," "merge," "create new," "upload," etc.
 - An action almost always occurs on the same page
 
+### Toggle Buttons
+
+Toggle buttons are used either to switch between two states, such as check and unchecked, or to trigger two actions like on or off.
+
+#### Toggle Buttons with Icons
+
+Icon buttons have an aria-label that will describe the action that the user will be performing when using screen readers.
+
+**Best Practices:**
+- Limit to 2-3 descriptive words.
+- Use verbs first to describe the action such as “Toggle” , “Favorite” or “Save”.
+- Use sentence case capitalization.
+- Words such as “the”, “a” be avoided.
+
+For Buttons with Icons ONLY, it is important that you show the two following elements:
+
+1. Show the current state of the button through the use of an icon that best represents the action.
+
+2. Inform the user of what is going to happen next by including `a11yTitle`, `aria-label`, or changing the button's `label`.
```

Comment:
Drop "that best represents the action", as it is a motherhood-and-apple-pie statement and offers no specific guidance.  It's a "Do your best" gee-up.  ie. every UI designer/developer will be making a personal choice about what best represents the action.

Drop ", or changing the buttons's `label`", since there is no label for "Buttons with Icons ONLY".  However, this needs to be captured in the section above. (I'll add a comment to that too).

I'll propose the following re-wording...

For Buttons with Icons ONLY, the icon shows current state and the aria-label describes the next state.  It is important that you show the two following elements:

1. Show the current state of the button through the use of an icon
2. Inform the user of what is going to happen next by including `a11yTitle` or `aria-label`.